elevate-cpanel

```diff
@@ -388,6 +388,10 @@ sub do_update ($self) {
     return 0;
 }
 
+sub base_url ($self) {
+    return $self->{'base_url'} ||= $ENV{'ELEVATE_BASE_URL'} || DEFAULT_ELEVATE_BASE_URL;
```

Comment:
not really sure how will behave the version compared with different BASE_URL... 
but ok

Reply:
Use case: We decide to change something about the way updates are detected and handled. We need to QA-test this before publishing to `release`. QA (or someone on their behalf) can create a branch off of `main` and both bump the version on this as well as create their own testing tag in place of `release` using `make RELEASE_TAG=tag release`. Before this change, one would need to hand-edit the script to point to this different URL, but now it can be specified during the actual test as an environment variable. (I could have made it an undocumented flag, I suppose.)

This mirrors the way that I tested the creation of the update process itself, by creating branch `test-make-release` and tag `test-tag`. (I've kept those around for now, but I'll delete them once we make the next release.)